### PR TITLE
README.md: Update "Tested up to" to 6.5

### PR DESCRIPTION
**Pull request by acicovic**
## Description
This PR updates the "Tested up to" field of `README.md` to `6.5`.

## Motivation and Context
Keep our `README.md` updated with the latest information.

## How Has This Been Tested?
- Current tests pass.
- No code changes.

## Summary by CodeRabbit

- **Documentation**
	- Updated the "Tested up to" version in the software package metadata from 6.4 to 6.5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Stable tag: 3.14.2  
 Requires at least: 5.2  
-Tested up to: 6.4  
+Tested up to: 6.5
```
[the remark this answers is not in the document]
Comment (acicovic) [reply README.md:5]: This suggestion is incorrect. This project is a WordPress plugin and contains a README.md with readme header information as specified in https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information. The text in question does not need grammar checking.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Stable tag: 3.14.2  
 Requires at least: 5.2  
-Tested up to: 6.4  
+Tested up to: 6.5  
 Requires PHP: 7.2  
 License: GPLv2 or later  
 License URI: https://www.gnu.org/licenses/gpl-2.0.html
```
[the remark this answers is not in the document]
Comment (acicovic) [reply README.md:8]: The proposed changes are out of scope of this PR.

**Comment by acicovic** [comment]
@coderabbitai resolve

**Review by vaurdan — APPROVED** [review]
Looks good! Let me know if you think we should disable CodeRabbit on .md/text files or if you think the feedback is (sometimes) useful.

**Comment by acicovic** [comment]
Hi, let's leave as-is for now.

Mixed feeling about it currently, since some of the feedback is useful, but it has made similar mistakes although it had feedback and had "learned" about them. If this trend continues, we might need to make adjustments.

**Pull request merged**